Release 0.10 (#2315)
scipp/scipp: docs/about/release-notes.rst

Release Notes
=============

v0.11.0 (unreleased)
--------------------

Features
~~~~~~~~

Breaking changes
~~~~~~~~~~~~~~~~

Bugfixes
~~~~~~~~

Deprecations
~~~~~~~~~~~~

Stability, Maintainability, and Testing
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Contributors
~~~~~~~~~~~~

Simon Heybrock :sup:`a`\ ,
Neil Vaytet :sup:`a`\ ,
Tom Willemsen :sup:`b, c`\ ,
and Jan-Lukas Wynen :sup:`a`

v0.10.0 (December 2021)
-----------------------

Features
~~~~~~~~

* Add support for auto-completions for slices and metadata access in Jupyter `#2304 <https://github.com/scipp/scipp/pull/2304>`_.

> * ``pip`` packages are now built, in addition to the existing builds for ``conda`` packages.

Reviewer: ```suggestion
* Start publishing ``pip`` packages to `PyPI <https://pypi.org/>`_ in addition to ``conda`` packages.
```
The old formulations sounds to me like we built packages once and that is it.

> Breaking changes

Reviewer: Remove empty section
